src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -1,6 +1,34 @@
 package com.sap.piper
 
 class ConfigurationHelper implements Serializable {
+    static class ConfigLoader implements Serializable {
+        private Map config
+        private String name
+
+        ConfigLoader(Script step){
+            if(!step.STEP_NAME)
+                throw new IllegalArgumentException('Step has no public name property!')
+            name = step.STEP_NAME
+            step.prepareDefaultValues()
+            config = ConfigurationLoader.defaultStepConfiguration(step, name)
+        }
+
+        ConfigLoader mixinStepConfig(stepWithEnv, Set filter = null){
+            Map stepConfiguration = ConfigurationLoader.stepConfiguration(stepWithEnv, name)
```

Comment:
only hand in `commonPipelineEnvironment` and use `ConfigurationLoader.stepConfiguration([commonPipelineEnvironment: cpe], name)`